Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -1,3 +1,6 @@
+# ignore
+--exclude Carthage,Pods,SourcePackages
```

Comment:
What is SourcePackages?

Reply:
In a Swift project / git repo, where is this directory? Is it at the repo root, or is it somewhere else?

Reply:
It is located at the root of the repository.